### DISCUSSION TITLE
Complicated color change

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [dlemstra/Magick.NET#1129] [Help]
With ColorFuzz we can make another color with near colors.

But can we do difficult changes.
From one group color to another? 
![from](https://user-images.githubusercontent.com/67605008/153774428-730f887c-6a65-4816-8ba6-5c3a3a5e8359.png)
![to](https://user-images.githubusercontent.com/67605008/153774430-0ef24b97-3f18-4903-943e-713a3d7ffd0f.png)

**Comment by dlemstra** [comment]
I might be better to ask your question here instead: https://github.com/ImageMagick/ImageMagick/discussions. And if you get an answer I can help you translate this to C#.

**Reply by jaka1-newbie** [reply]
Thx. Try there.
I think - may be we can grayscale 1 image - make from it mask - and compare with new color...

**Comment by jaka1-newbie** [comment]
https://github.com/ImageMagick/ImageMagick/discussions/4840

**Reply by dlemstra** [reply]
Happy to see that you got an answer. I probably won't be able to translate the code before Friday. Would you mind giving it a try first yourself and ask for help here when you need it? Most options on the comand line are methods of the `MagickImage` class.
